doc/lib_term.md

```diff
@@ -23,7 +23,7 @@ Here is an example clock display from the examples folder, using
 the char_circle and char_line primitives to draw the clock face
 and the hands:
 
-![example image showing a clock](example_lib_term.png)
+![example image showing a clock](lib_term_illustration.png)
```

Comment:
I dislike this rename.  The original name made it clear that this is an image connected to the one example program that USES lib_term (example_lib_term.ks), rather than to lib_term itself.

lib_term is not a tool for making clocks.  The clock is just what example_lib_term.ks does.